modules/calib3d/test/test_cameracalibration.cpp

```diff
@@ -812,7 +813,6 @@ void CV_CameraCalibrationTest_CPP::calibrate(Size imageSize,
     {
         Mat r9;
         cvtest::Rodrigues( rvecs[i], r9 );
-        cv::transpose(r9, r9);
```

Comment:
added by @vpisarev here #13081

https://github.com/opencv/opencv/pull/13081/files#diff-3122fe7397a59cc808249e6c0ac6e1a089997df71021aade6778799f59b925f7R815

There is not such line on 3.4 branch.